sanitizers/src/test/java/com/example/BUILD.bazel

```diff
@@ -25,6 +25,20 @@ java_fuzz_target_test(
     target_class = "com.example.ReflectiveCall",
 )
 
+java_fuzz_target_test(
+    name = "ReflectiveCallDisabled",
+    srcs = [
+        "ReflectiveCallDisabled.java",
+    ],
+    fuzzer_args = [
+        "-runs=0",
+    ] + select({
+        "@platforms//os:windows": ["--disabled_hooks=com.example.DisabledHook;com.code_intelligence.jazzer.sanitizers.ReflectiveCall"],
```

Comment:
Since this is a pain and class names never contain `,`, how about using that as a separator? I see relying on `;` or `:` for anything other than `--cp` as a mistake, but back in the day we just didn't think about it as Windows support wasn't on the roadmap at all. For new features, I would like to avoid repeating this mistake.

Reply:
Changed to use commas to separate elements of disabled sanitizers

Reply:
I've removed this change for now and will create a dedicated PR for changing the separator